package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "dynamoose",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

This needs to be reverted. This (along with `package-lock.json` version) will be bumped automatically as part of our automated release process.

```suggestion
  "version": "2.3.0",
```

---

Ah gotcha. Sorry if I overlooked that somewhere!